### PR TITLE
removed minigallery

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes https://github.com/sunpy/sunpy/issues/4167

### CODE REVIEW

**Comment by nabobalis** [comment]
The last commit should fail the RTD build and we should see it. Hopefully. 
It works locally for me.

**Comment by nabobalis** [comment]
Everything will fail but the doc failing is the important check.

**Comment by nabobalis** [comment]
I am wondering if we need to bring back a circleci docs build only to check for warnings.

**Comment by nabobalis** [comment]
It worked?!
https://readthedocs.org/api/v2/build/11484571.txt

File: examples/computer_vision_techniques/loop_edge_enhance.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 aia = sunpy.map.Map(AIA_171_IMAGE)
 bottom_left = SkyCoord(750 * u.arcsec, -200 * u.arcsec, frame=aia.coordinate_frame)
 top_right = SkyCoord(1500 * u.arcsec, 550 * u.arcsec, frame=aia.coordinate_frame)
-aia_smap = aia.submap(bottom_left, top_right=top_right)
+aia_smap = aia.submap(bottom_left, top_right)
```
Comment (nabobalis) [inline examples/computer_vision_techniques/loop_edge_enhance.py:24]: It seems the warning is not captured or raised if there are more than subgroup of examples run.

**Comment by nabobalis** [comment]
> It worked?!
> [readthedocs.org/api/v2/build/11484571.txt](https://readthedocs.org/api/v2/build/11484571.txt)

So it worked when all I ran was the `computer_vison` set of examples but as soon as I add the rest back, it gets ignored.

**Comment by nabobalis** [comment]
I was unable to get the build to fail on a regular basis, I have removed it from this PR.

File: docs/guide/plotting.rst [hunk 2]
```diff
@@ -288,8 +288,7 @@ units to plot a AIA image and a zoomed in view of an active region.
                            frame=smap.coordinate_frame)
     top_right = SkyCoord(x0 + length, y0 + length,
                          frame=smap.coordinate_frame)
-    submap = smap.submap(bottom_left, top_right)
-
+    submap = smap.submap(bottom_left, top_right==top_right)
```
Comment (Cadair) [inline docs/guide/plotting.rst:291]:
```suggestion
    submap = smap.submap(bottom_left, top_right=top_right)
```

Comment (nabobalis) [reply docs/guide/plotting.rst:291]: Fixed.

**Comment by nabobalis** [comment]
> Sure, what's the beef with the RTD build?

Trying to get the SunpyDepWarning to error the build.

**Comment by Cadair** [comment]
Ah right.

**Pull request merged**